backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java

```diff
@@ -48,10 +55,21 @@
   /**
    * Constructs an object which implements the StoryAnalysisManager
    * class using the Google Perspective API.
+   *
+   * @throws APINotAvailableException when it can't create an instance of the PerspectiveAPI
+   *    (this most likely occurs if the "PerspectiveAPIKey.java" file is not present)
    */
-  public PerspectiveManager() throws NoAppropriateStoryException {
-    // TODO: instantiate the PerspectiveAPI for this instance
-    perspectiveAPI = null;
+  public PerspectiveManager() throws APINotAvailableException {
```

Comment:
FYI: For future reference, the link between PerspectiveManager and StoryAnalysisManager would be clearer if this class was named `PerspectiveStoryAnalysisManager` ("a story analysis manager implemented using Perspective").

As is, it's not obvious that PerspectiveManager is related to StoryAnalysisManager unless you check the interface.

Reply:
I changed the name, so it should hopefully be clearer.

Reply:
Done